src/models/sqlOutputContentProvider.ts

```diff
@@ -120,6 +120,16 @@ export class SqlOutputContentProvider {
         this.openLink(content, columnName, linkType);
     }
 
+    public copyHeadersRequestHandler(
+        uri: string,
+        batchId: number,
+        resultId: number,
+    ): void {
+        void this._queryResultsMap
```

Comment:
does this need to handle the case where the map doesn't contain `uri`?  just some null propagations